Base/usr/share/man/man5/af.md

```diff
@@ -1,18 +1,49 @@
 ## Name
 
-af - Application File format
+af - Application File format (.af)
 
 ## Synopsis
 
 The Application Files define System Menu entries and launcher file types / protocols.
 
 ## Description
 
-.af files are human-readable and are a subset of the INI-format, have no easily detectable filemagic. These files define System Menu entries and launcher file types / protocols.
+Application files are a subset of the INI format.
+They have no easily detectable filemagic and contain application information (App group):
 
-They are stored in [`/res/apps`](../../../../res/apps).
+| description                      | key           |
+|----------------------------------|---------------|
+| name                             | Name          |
+| executable path                  | Executable    |
+| category (optional)              | Category      |
+| description (optional)           | Description   |
+| application icon path (optional) | IconPath      |
+| run in terminal flag (optional)  | RunInTerminal |
 
-## See Also
+and launcher information (Launcher group, optional):
 
+| description                           | key       |
+|---------------------------------------|-----------|
+| supported file types separated by ',' | FileTypes |
+| protocols separated by ','            | Protocols |
+
+-------------------------------------------------------
+All application files are stored in **read-only** memory in [`/res/apps`](../../../../res/apps).
```

Comment:
I wouldn't specifically mention that /res/apps is mounted read-only, and the term "read-only memory" is not something we use for non-volatile storage. After all, this might change in the future when you want to install apps on-the-fly.

Reply:
Would it be better to replace "read-only" with "non-volatile" or does it make sense to omit such details altogether?

Reply:
Just omit it.